### PR TITLE
Moved to Swift 5.10.0

### PR DESCRIPTION
**Pull request by mstokercricut**

### CODE REVIEW

**Review by dvosscricut — COMMENTED** [review]
@mstokercricut  it looks like docker is failing on ubuntu and windows integration tests:

**Review by rodzikkonrad — APPROVED** [review]

**Comment by rodzikkonrad** [comment]
@mstokercricut do you plan to write more tests to have the same code coverage percentage or you will drop the percentage down to be able to merge that PR?

File: kotlin-runtime/build.gradle.kts [hunk 1]
```diff
@@ -133,7 +133,7 @@ tasks {
 
 dependencies {
     implementation(kotlin("stdlib:1.9.10"))
-    implementation("com.cricut:android-swift-runtime:1.1.0")
+    implementation("com.cricut:android-swift-runtime:1.2.0-alpha7")
```
Comment (cobbal) [inline kotlin-runtime/build.gradle.kts]: This should probably not be merged to main

**Review by cobbal — DISMISSED** [review]
Looks fine to me, but android-swift-runtime should get a real release before this is merged

File: .github/workflows/iota-runtime.yaml [hunk 2]
```diff
@@ -159,7 +159,8 @@ jobs:
           set -euxo pipefail
 
           mkdir -p "$LIB_DIR"
-          cp -R "/c/Program Files/Swift/runtime-development/usr/bin/"*.dll "$LIB_DIR"
+          SWIFT_RUNTIME_DIR=`realpath "$SDKROOT/../../../.." | sed s#Platforms#Runtimes#g`
```
Comment (cobbal) [inline .github/workflows/iota-runtime.yaml]: Windows is known for putting spaces in paths, probably best to quote here
```suggestion
          SWIFT_RUNTIME_DIR="$(realpath "$SDKROOT/../../../.." | sed s#Platforms#Runtimes#g)"
```

File: integration-tests/TestAPI-bindings/node-test/Protocols.test.ts [hunk 3]
```diff
@@ -262,7 +264,8 @@ async function testAsyncForeignSideFunctionsCore(a: TestAPI.TestAsyncFunctions)
             return (fl / d) + (9 * l);
         }
     );
-    expect(j).toEqual("18.227272727272727");
+    // TODO: Fix https://cricut.visualstudio.com/Cricut/_workitems/edit/484036
```
Comment (dvosscricut) [inline integration-tests/TestAPI-bindings/node-test/Protocols.test.ts]: @mstokercricut This should be fixed with FishyJoes 4.7.11 now

you can put all the tests back

File: kotlin-runtime/build.gradle.kts (hunk 1 above)
Comment (dvosscricut) [reply kotlin-runtime/build.gradle.kts]: updated to 1.2.0 now @cobbal

File: integration-tests/TestAPI-bindings/node-test/Protocols.test.ts (hunk 3 above)
Comment (dvosscricut) [reply integration-tests/TestAPI-bindings/node-test/Protocols.test.ts]: done!

**Review by dvosscricut — APPROVED** [review]

**Pull request merged**